frontend/components/buttons/Button/_styles.scss

```diff
@@ -320,4 +330,14 @@ $base-class: "button";
     display: flex;
     justify-content: space-between;
   }
+
+  .transparent-text {
+    opacity: 0;
+  }
+
+  .children-wrapper {
```

Comment:
{children} is required to be the only child of the parent component, so we needed to move the styling onto this new div to ensure the icon aligned with the text